target/plantbattles-1/style.css

```diff
@@ -0,0 +1,33 @@
+/* -- ALL PAGES -- */
+html{
+    background-image: url("images/background.png");
+    background-position: center;
+    background-size: cover;
+}
```

Comment:
same with multiple copies of this file. probably fine to delete whichever copy we don't need.